README.md

```diff
@@ -93,14 +94,24 @@ Generate application key
 
 Install npm/yarn dependencies  (Preference is using **Yarn**)
 ```bash
-npm install or yarn install 
+npm install or yarn install
 ```
 
 
 This command will help migrate the database and populate the database!
 ```bash
 php artisan migrate --seed
 ```
+If an error of permission denied occurs in MySQL, try creating a non-root user with a password and update the .env file as required. Also make sure that the database name provided in the .env file, actually exists in the MySQL databases. You can find detailed solution [here](https://stackoverflow.com/questions/46680785/laravel-artisan-access-denied-for-rootlocalhost-on-console-only/47116614#47116614)
+
+After configuration, the .env file should look somewhat similar like this
+```DB_CONNECTION=mysql
```

Comment:
Please submit one PR for one issue. Also I am curious what editor do you use, I get a bunch of changes that are not related... I thought I included an `editorconfig` sometime ... Will check ...